insights/specs/__init__.py

```diff
@@ -803,6 +803,7 @@ class Specs(SpecSet):
     sssd_conf_d = RegistryPoint(multi_output=True)
     sssd_logs = RegistryPoint(multi_output=True, filterable=True)
     sys_block_queue_stable_writes = RegistryPoint(multi_output=True)
+    sys_block_queue_max_segment_size = RegistryPoint(multi_output=True)
```

Comment:
The order of these two specs are not in alphabet order, would you mind moving both of them correctly under the `swift_proxy_server_conf`? Thanks.